src/components/EnvironmentDeployments/EnvironmentDeployments.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { EnvDeployment } from '../../api/types';
+import { TableContainer, Typography, makeStyles } from '@material-ui/core';
+import { Table } from '@backstage/core-components';
+import { defaultDeploymentColumns } from './columns';
+import GitHubIcon from '@material-ui/icons/GitHub';
+
+const useStyles = makeStyles(theme => ({
+  empty: {
+    padding: theme.spacing(2),
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+/** @public */
+export const EnvironmentDeployments = (props: {
+  environment: string;
+  loading: boolean;
+  currentDeployments: EnvDeployment[];
+  projectSlug: string;
+}) => {
+  const [currentDeployments, setCurrentDeployments] = useState<EnvDeployment[]>(
+    [],
+  );
+  const classes = useStyles();
+
+  useEffect(() => {
+    if (!props.loading) {
+      setCurrentDeployments(props.currentDeployments);
+    }
+  }, [props.loading, props.currentDeployments]);
+
+  for (let i in currentDeployments) {
+    (currentDeployments[i] as any).projectSlug = props.projectSlug;
+  }
+  currentDeployments.sort((a, b) => (a.id < b.id ? 1 : -1));
+  const display = (
+    <TableContainer>
+      <Table
+        title={'Deployments: ' + props.environment}
+        columns={defaultDeploymentColumns}
+        options={{
+          header: true,
+          padding: 'dense',
+          paging: true,
+          pageSize: 10,
+          search: false,
+          showEmptyDataSourceMessage: true,
+          toolbar: true,
+          tableLayout: 'auto',
+        }}
+        data={currentDeployments}
+        isLoading={props.loading}
+        emptyContent={
+          <div className={classes.empty}>
+            <Typography variant="body1">
+              No deployments for <code>{props.projectSlug}</code> in{' '}
+              <code>{props.environment}</code> environment.
+            </Typography>
+          </div>
+        }
+        actions={[
+          {
+            icon: () => <GitHubIcon />,
+            tooltip: 'Environment Deployments',
+            isFreeAction: true,
+            onClick: () => {
+              window.open(
+                `https://github.com/${props.projectSlug}/deployments/${props.environment}`,
```

Comment:
_This can be fixed in a later PR._ The generated GitHub URL doesn't work when `props.environment` is `"Any"`. We also ought to see if there's a standard way to have the domain part come from configuration... similar to how the new API in this plugin doesn't have "github.com" anywhere in its source.

Reply:
I'll try to come up with a better way to do this.  This link won't show up for a environemnt of "Any", but I see what you mean about this part.

Reply:
Well... [I saw the link](https://techhub.uat.mktp.io/catalog/default/component/applications-and-policies-mfe/deployments) for an environment of "Any" which is why I reported it.

Reply:
Oh wait.  I just realized this was for the link at the top right.  I'll clean this up.